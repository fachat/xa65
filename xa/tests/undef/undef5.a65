
	.text


	.(

	.(

	lda l1

	.)
l1
	.)

	lda l1

