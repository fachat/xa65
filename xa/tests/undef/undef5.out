00000000  01 00 6f 36 35 00 00 00  00 10 06 00 00 04 00 00  |..o65...........|
00000010  00 40 00 00 04 00 00 00  00 00 00 ad 03 10 ad 00  |.@..............|
00000020  00 01 00 6c 31 00 02 82  03 80 00 00 00 00 00 00  |...l1...........|
00000030
